src/shared/models/asset/asset.service.ts

```diff
@@ -15,8 +15,8 @@ export class AssetService {
   constructor(private assetRepo: AssetRepository) {}
 
   async getAllAsset(blockchains: Blockchain[]): Promise<Asset[]> {
-    blockchains ??= [Blockchain.DEFICHAIN];
-    return this.assetRepo.find({ where: { blockchain: In(blockchains) } });
+    const where = blockchains.length > 0 ? { blockchain: In(blockchains) } : undefined;
+    return where ? this.assetRepo.find({ where: where }) : this.assetRepo.find();
```

Comment:
```ts
return blockchains.length > 0 ? this.assetRepo.find({ where: { blockchain: In(blockchains) } }) : this.assetRepo.find();
```
Looks simpler to me 😆

Reply:
true. initially I thought if where is undefined, it would automatically default back to .find(), but it doesn't and then I didn't change it around again